packages/fast-usdc/src/exos/advancer.js

```diff
@@ -212,15 +212,22 @@ export const prepareAdvancerKit = (
          * @param {Error} error
          * @param {AdvancerVowCtx & { tmpSeat: ZCFSeat }} ctx
          */
-        onRejected(error, { tmpSeat }) {
-          // TODO return seat allocation from ctx to LP?
-          log('🚨 advance deposit failed', q(error).toString());
-          // TODO #10510 (comprehensive error testing) determine
-          // course of action here
+        onRejected(error, { tmpSeat, advanceAmount, ...restCtx }) {
+          // we don't expect this to be a common failure. if it happens, return funds to LP
```

Comment:
Is how common it is relevant? We're talking user funds here, after all.

Reply:
Good point, updated the comment with "how"